### PR TITLE
Add multi-model-server and model-archiver

### PR DESCRIPTION
**Pull request by sodre**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/model-archiver/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+build:
+  script: {{ PYTHON }} -m pip install ./model-archiver -vv
+  number: 0
+  entry_points:
+    - model-archiver = model_archiver.model_packaging:generate_model_archive
+  skip: True  # [win or osx]
+  
+requirements:
+  host:
+    - python >=3.6
```
Comment (sodre) [inline recipes/model-archiver/meta.yaml:24]:
```suggestion
    - python
```

File: recipes/model-archiver/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    - pip
+  run:
+    - python >=3.6
```
Comment (sodre) [inline recipes/model-archiver/meta.yaml:27]:
```suggestion
    - python
```

File: recipes/multi-model-server/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  entry_points:
+    - multi-model-server = mms.model_server:start
+    - multi-model-export = mms.export_model:main
+  skip: True  # [win or osx]
+
+requirements:
+  build:
+    - openjdk 8.*
+  host:
+    - python >=3.6
```
Comment (sodre) [inline recipes/multi-model-server/meta.yaml:26]:
```suggestion
    - python
```

File: recipes/multi-model-server/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    - pip
+  run:
+    - python >=3.6
```
Comment (sodre) [inline recipes/multi-model-server/meta.yaml:29]:
```suggestion
    - python
```

**Review by sodre — COMMENTED** [review]
fix lint issues

**Comment by sodre** [comment]
@conda-forge/help-python, this recipe is ready for review. Thank you

**Comment by sodre** [comment]
Close #16507

**Comment by sodre** [comment]
@conda-forge/staged-recipes model-server recipe is ready for review.

File: recipes/model-archiver/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  sha256: 64876dbfd56671d34aedc40ec8de7988bd3ed25c3806cd463aaa62c094890ab8
+  patches:
+    - skip-tests.patch
+
+build:
+  script: {{ PYTHON }} -m pip install ./model-archiver -vv
+  number: 0
+  entry_points:
+    - model-archiver = model_archiver.model_packaging:generate_model_archive
+  skip: True  # [win or osx]
```
[the remark this answers is not in the document]
Comment (sodre) [reply recipes/model-archiver/meta.yaml:20]: They can be converted to no-arch after the recipe is merged. 

During staged-recipes build, the macOS case throws an unexpected [exception](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=686934&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=1272). 

Windows support is only [experimental](https://github.com/awslabs/multi-model-server#prerequisites) upstream.

[the remark this answers is not in the document]
Comment (ocefpaf) [reply recipes/model-archiver/meta.yaml:20]: > During staged-recipes build, the macOS case throws an unexpected [exception](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=686934&view=logs&j=e35d4f76-8ff2-5536-d795-df91e63eb9f7&t=fa7b4b17-b6ff-5c9c-8cfc-15f888c92310&l=1272).

Looks like it is not support but that exception is a bit cryptic.

**Pull request merged**